extensions/master_extension_prepull/v1/win-e2e-master-extension.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+echo "start master extension" >> /tmp/master_extension.log
+
+KUBECONFIG="$(find /home/*/.kube/config)"
+KUBECTL="kubectl --kubeconfig=${KUBECONFIG}"
+
+# Wait for coredns pod to be ready
+while true; do
+	COREDNS_READY=$(${KUBECTL} get pods -l k8s-app=kube-dns -n kube-system -o custom-columns=STATUS:status.containerStatuses[0].ready --no-headers)
+	if [ "${COREDNS_READY}" = "true" ]; then
+		echo "$(date -R) - coredns is ready" >> /tmp/master_extension.log
+		${KUBECTL} get pods --all-namespaces >> /tmp/master_extension.log
+		break
+	fi
+	sleep 2
+done
```

Comment:
Is there away to avoid duplicating this logic for waiting on kubesystem pods to be ready?

Reply:
Since coredns pods are deployed to kube-system, I converted the for loop to `kubectl wait --for=condition=ready pod --all -n kube-system --timeout 10m`